src/dev/tsconfig.json

```diff
@@ -40,5 +40,7 @@
     "@kbn/import-locator",
     "@kbn/config-schema",
     "@kbn/journeys",
+    "@kbn/core-test-helpers-so-type-serializer",
```

Comment:
I'm not sure if the placement of the script in `src/dev` is the best, is it okay to pull in these dependencies because of it?

Reply:
The placement seems okay. We could always move SO migration snapshot into a package if its needed for some reason in the future.